packages/jest/src/migrations/update-8-3-0/update-8-3-0.ts

```diff
@@ -15,18 +15,23 @@ function updateJestConfig(host: Tree) {
       ts.ScriptTarget.Latest
     );
     const changes: RemoveChange[] = [];
-
-    getSourceNodes(sourceFile).forEach(node => {
+    const sourceNodes = getSourceNodes(sourceFile);
+    sourceNodes.forEach((node, index) => {
       if (
         ts.isPropertyAssignment(node) &&
         ts.isIdentifier(node.name) &&
         node.name.text === 'collectCoverage'
       ) {
+        const expectedCommaNode = sourceNodes[index + 4];
+        const isFollowedByComma =
+          expectedCommaNode.kind === ts.SyntaxKind.CommaToken;
         changes.push(
           new RemoveChange(
             'jest.config.js',
             node.getStart(sourceFile),
-            node.getFullText(sourceFile)
+            isFollowedByComma
+              ? node.getFullText(sourceFile)
+              : node.getText(sourceFile)
```

Comment:
This is to workaround a problem in: https://github.com/nrwl/nx/blob/c213ae1a999d7d4567be70afb97fb79247fe8744/packages/workspace/src/utils/ast-utils.ts#L362

The problem is:
- For Replace changes: will always work fine, we can just replace all the node's text with something else, we will keep any symbols before and after
- for Insert changes: it should work fine again, because we can decide **where** we want to add it, and thus whether we need to add commas/other tokens
- **for Removal changes** there is a problem: we do not know where the text being removed is in the file..it might be part of a comma-delimited list, it might be at the end of a list and followed by a closing bracket etc. So we need to determine how to modify the file **around** our removal to avoid breaking it

---

In this specific instance, if we wanted to remove the `passWithNoTests: true` option in a file that looked like:

```javascript
module.exports = {
  testMatch: ['**/+(*.)+(spec|test).+(ts|js)?(x)'],
  transform: {
    '^.+\\.(ts|js|html)$': 'ts-jest'
  },
  resolver: '@nrwl/jest/plugins/resolver',
  moduleFileExtensions: ['ts', 'js', 'html'],
  coverageReporters: ['html'],
  passWithNoTests: true
};
```
...our utility would actually remove the ending brace as well, because it's configured to remove surrounding characters as well
...if we modified our utility to not remove any surrounding characters, then it would work **in this specific** case, but it would not work if the option was in the **middle** of the file - because it would need to remove any leftover commas!